src/main/play-doc/operation/Install.md

```diff
@@ -980,7 +719,7 @@ By default ConductR will be deployed with a single instance.
 
 Wait until the ConductR instance's health to marked as healthy before proceeding.
 
-If more instances is required, scale up to the desired number of instances, and ensure each of the ConductR instance's health to marked as healthy before proceeding. Each ConductR instance will attempt to register itself with a separate framework id, and hence it is expected to have multiple ConductRs appearing in the list of application. These ConductR instances will attempt to form a cluster of its own as part of the startup.
+If more instances is required, scale up to the desired number of instances, and ensure each of the ConductR instance's health to marked as healthy before proceeding. These ConductR instances will attempt to form a cluster of its own as part of the startup.
```

Comment:
If more instances are required, scale the application to the desired total and ensure that all of the instances are healthy before proceeding.